Tests/FoundationEssentialsTests/ProcessInfoTests.swift

```diff
@@ -94,13 +94,12 @@ final class ProcessInfoTests : XCTestCase {
         XCTAssertTrue(memory > 0, "ProcessInfo doesn't think we have any memory")
     }
 
-    func testSystemUpTime() {
+    func testSystemUpTime() async throws {
         let now = ProcessInfo.processInfo.systemUptime
         XCTAssertTrue(
             now > 1, "ProcessInfo returned an unrealistically low system uptime")
         // Sleep for 0.1s
-        var ts: timespec = timespec(tv_sec: 0, tv_nsec: 100000000)
-        nanosleep(&ts, nil)
+        try await Task.sleep(for: .milliseconds(1000))
```

Comment:
nit (sorry): the comment says "0.1s" but the code sleeps for 1.0s, should we update one or the other to be consistent?

Reply:
No, that should've been 100, not 1000.